src/main/java/duke/Event.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Inherits from the Task class. Represents a event happening at a certain time.
+ * Event object stores task description, done status, date of event, start and end time of event.
+ * A Event object is marked as not done by default.
+ * A Event object is able to be set as done by calling setDone() method from the parent class.
+ */
+public class Event extends Task {
+    protected LocalDate at;
+    protected LocalTime start;
+    protected LocalTime end;
+
+    /**
+     * Event object constructor overrides Task constructor.
+     * Takes in date of event, start and end time of event in addition to the task description
+     * and parses into LocalDate and LocalTime format.
+     * @param description description of task
```

Comment:
Maybe an empty line between description and param section?

Reply:
Resolved this. Thank you!